Tests/js/views/services/assets/ez-roleserversideviewservice-tests.js

```diff
@@ -632,8 +633,61 @@ YUI.add('ez-roleserversideviewservice-tests', function (Y) {
         }
     });
 
+    refreshAssignmentsTabTest = new Y.Test.Case({
+        name: "eZ Role Server Side View Service refresh assignments tab test",
+
+        setUp: function () {
+            this.capi = new Mock();
+            this.app = new Mock();
+            this.roleId = 42;
+
+            this.service = new Y.eZ.RoleServerSideViewService({
+                capi: this.capi,
+                app: this.app,
+            });
+
+            Mock.expect(this.service, {
+                method: 'get',
+                args: [Mock.Value.String],
+                returns: this.app,
+            });
+            Mock.expect(this.service, {
+                method: 'load',
+                args: [Mock.Value.Function],
+            });
+            Mock.expect(this.app, {
+                method: 'get',
+                args: [Mock.Value.String],
+            });
+        },
+
+        tearDown: function () {
+            delete this.capi;
+            delete this.app;
+            delete this.service;
+        },
+
+        "Should refresh the page with tabId parameter added": function () {
+            var data = {
+                    roleId: this.roleId,
+                };
+
+            Mock.expect(this.app, {
+                method: 'set',
+                args: ['loading', true],
+            });
+            Mock.expect(this.app, {
```

Comment:
It's not checking if `navigateTo` method was called. It only checks whether the parameters passed to the function are those you are expecting, but if the method wasn't called at all then test wouldn't catch that. To check if the method was called you should do:
`Mock.verify(this.app);` at the end of the test
OR
provide `run` attribute  where you will switch the flag `appNavigatedToCalled` to `true` then make assertion for it at the end.